packages/wdio-utils/utils.d.ts

```diff
@@ -0,0 +1 @@
+export * from './src/index'
```

Comment:
we don't push `src` files to NPM

Reply:
I don't quite understand what you mean... I had to do this because else it was always complaining about missing type declarations... Maybe you have a better suggestion here?

Reply:
you should have types now if you rebase on master